<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Editor</class>
 <widget class="QWidget" name="Editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Sigmet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="headGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Head</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="3">
       <widget class="QComboBox" name="description"/>
      </item>
      <item row="1" column="7">
       <widget class="QLineEdit" name="obsTime">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="beginningTime"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="sequence"/>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="obsTimeLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>OBS Time</string>
        </property>
        <property name="buddy">
         <cstring>obsTime</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="forecastLabel">
        <property name="text">
         <string>Forecast</string>
        </property>
        <property name="buddy">
         <cstring>forecast</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string notr="true">Name</string>
        </property>
        <property name="buddy">
         <cstring>name</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="descriptionLabel">
        <property name="text">
         <string>Phenomena Description</string>
        </property>
        <property name="buddy">
         <cstring>description</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="beginningTimeLabel">
        <property name="text">
         <string>Beginning Time</string>
        </property>
        <property name="buddy">
         <cstring>beginningTime</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="sequenceLabel">
        <property name="text">
         <string>Sequence</string>
        </property>
        <property name="buddy">
         <cstring>sequence</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QLineEdit" name="name"/>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="phenomenaLabel">
        <property name="text">
         <string>Phenomena</string>
        </property>
        <property name="buddy">
         <cstring>phenomena</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QComboBox" name="forecast"/>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="phenomena"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="endingTime"/>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="endingTimeLabel">
        <property name="text">
         <string>Ending Time</string>
        </property>
        <property name="buddy">
         <cstring>endingTime</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>beginningTime</tabstop>
  <tabstop>endingTime</tabstop>
  <tabstop>sequence</tabstop>
  <tabstop>description</tabstop>
  <tabstop>phenomena</tabstop>
  <tabstop>name</tabstop>
  <tabstop>forecast</tabstop>
  <tabstop>obsTime</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
